homework/java/shelest_vladimir/login_server/src/test/java/by/jrr/service/IUserTest.java

```diff
@@ -0,0 +1,20 @@
+package by.jrr.service;
+
+import org.junit.Assert;
+import org.junit.Test;
+import by.jrr.bean.User;
+
+public class IUserTest {
```

Comment:
By default interface shouldn't have methods with body, so should be no case to test.